### ISSUE TITLE
Idea: Ability to "choose" time frame (by reading dump?).

### ISSUE THREAD

**Issue by filiphl** [ComputationalPhysics/atomify-lammps#281]
When visualizing, it would be practical to have the ability of jumping back to a previous frame. For instance if one notices anomalous behavior, it's of interest to find when and where it occurred. Typically visualization software would read the dump-files, which were stored during a run. Having the option of "looking" through dumps might be a way to go?

**Comment by andeplane** [comment]
I agree. While we won't try to replace Ovito as a post processing tool, it would be nice to open existing dumps. To be able to go back and forth, we should probably create a new time integrator in LAMMPS so all fixes and computes are evaluated correctly. Alternatively we can just apply a small script with `read_dump` commands and do `run 0` to evaluate everything. 

The problem with this is that some fixes/computes may be angry if we suddenly are going back in time...
